pyracing/response_objects/session_data.py

```diff
@@ -137,66 +138,67 @@ def __init__(self, dict):
             self.pos_finish = dict['finishpos']
             self.pos_finish_class = dict['finishposinclass']
             self.pos_start = dict['startpos']
-            self.reason_out = dict['reasonout']
+            self.reason_out = parse_encode(dict['reasonout'])
             self.reason_out_id = dict['reasonoutid']
-            self.restrict_results = dict['restrictresults']
-            self.sim_ses_name = dict['simsesname']
+            self.restrict_results = parse_encode(dict['restrictresults'])
+            self.sim_ses_name = parse_encode(dict['simsesname'])
             self.sim_ses_num = dict['simsesnum']
-            self.sim_ses_type_name = dict['simsestypename']
+            self.sim_ses_type_name = parse_encode(dict['simsestypename'])
             self.sub_level_new = dict['newsublevel']
             self.sub_level_old = dict['oldsublevel']
-            self.suit_color_1 = dict['suit_color1']
-            self.suit_color_2 = dict['suit_color2']
-            self.suit_color_3 = dict['suit_color3']
+            self.suit_color_1 = parse_encode(dict['suit_color1'])
+            self.suit_color_2 = parse_encode(dict['suit_color2'])
+            self.suit_color_3 = parse_encode(dict['suit_color3'])
```

Comment:
Why are colors encoded? What do they look like?

Reply:
hm, I just missed those. They are strings though

Reply:
Ahh, those were missed when removing. 
```suggestion
            self.suit_color_3 = dict['suit_color3']
```